go.mod

```diff
@@ -30,9 +30,11 @@ require (
 	github.com/shurcooL/vfsgen v0.0.0-20180825020608-02ddb050ef6b
 	github.com/stretchr/testify v1.3.0
 	github.com/xlab/treeprint v0.0.0-20180616005107-d6fb6747feb6
-	golang.org/x/net v0.0.0-20181201002055-351d144fa1fc
-	golang.org/x/text v0.3.1-0.20180805044716-cb6730876b98 // indirect
-	golang.org/x/tools v0.0.0-20190118193359-16909d206f00
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
```

Comment:
From `github.com/miekg/dns`, this repository hasn't yet moved to Go modules but it's in progress (https://github.com/miekg/dns/pull/980).